projects/igniteui-angular/src/lib/grids/filtering/base/grid-filtering-row.component.ts

```diff
@@ -498,6 +498,14 @@ export class IgxGridFilteringRowComponent implements AfterViewInit {
         this.value = value;
     }
 
+    /** @hidden @internal */
+    public inputGroupPrefixClick(event: MouseEvent, target: any) {
+        event.stopPropagation();
+        // TODO: try to get the target from the event
```

Comment:
There is a leftover TODO.

Reply:
I was not sure, in the scope of filter row, which element should be taken. This is why I added this TODO for guys implementing the filtering to fix or remove it.

Reply:
done